drivers/sensor/npm1300_charger/npm1300_charger.c

```diff
@@ -342,6 +376,31 @@ static int npm1300_charger_attr_set(const struct device *dev, enum sensor_channe
 		}
 		return mfd_npm1300_reg_write(config->mfd, CHGR_BASE, CHGR_OFFSET_EN_SET, 1U);
 
+	case SENSOR_CHAN_CURRENT:
+		if (attr != SENSOR_ATTR_CONFIGURATION) {
+			return -ENOTSUP;
+		}
```

Comment:
how about moving this outside of the case? then you won't have to repeat it every time... same in the attr_get

Reply:
I'll do that thanks.  attr_get needs to keep them separate as the attributes are different.